README.md

```diff
@@ -590,6 +590,8 @@ If you're using the `X-SourceMap` header instead, you can just omit `sourceMap.u
 
 - `evaluate` -- attempt to evaluate constant expressions
 
+- `arrows` -- compress and/or convert to arrow functions
```

Comment:
`arrows` (default `true`) -- convert ES5 style anonymous function expressions to arrow functions if permissible by language semantics. Note: `arrows` requires that the `ecma` compress option is set to `6` or greater.